src/ui/MessageContent/index.tsx

```diff
@@ -79,10 +77,7 @@ export default function MessageContent({
   const [mouseHover, setMouseHover] = useState(false);
   const [supposedHover, setSupposedHover] = useState(false);
 
-  const isByMe: boolean = isPendingMessage(channel, message as UserMessage | FileMessage)
```

Comment:
For now, just use 
```
  const isByMe = (userId === sender.userId)
    || (message.requestState === 'pending') // this part is important...
    || (message.requestState === 'failed'); // otherwise pending and failed messages will be treated as incoming
```

I recommend to do the refactor in `src/utils/index.ts` in separate PR and release later to reduce complexity of this PR